README.md

```diff
@@ -113,6 +113,7 @@ make check_tools
 | [markdown-link-check](https://www.npmjs.com/package/markdown-link-check) | Validates hyperlinks in Markdown files. |
 | [spectral](https://www.npmjs.com/package/@stoplight/spectral) | Validates OpenAPI 3.0 specification files. |
 | [swagger-cli](https://www.npmjs.com/package/swagger-cli) | Validates OpenAPI 3.0 specification files. Also bundles multiple OAS3 files into a single file, that is required by some downstream use-cases. |
```

Comment:
just checking that we need `swagger-cli` also if `redocly` does the same thing based on this explanation?

Reply:
Great question! Yes, for two reasons:

1. The clinic service uses `swagger-cli` to bundle its spec as a pre-process step before running codegen tools. I didn't want to rock that boat. It is used in `./scripts/generate_clinic.sh` to test that the generation works.
2. Because of #1, I also left it in as an extra linter/validation tool to ensure that no API spec changes break the clinic service builds. It is used in `./scripts/check_spec.sh`